librz/il/il_vm_eval.c

```diff
@@ -342,3 +419,25 @@ RZ_API bool rz_il_evaluate_effect(RZ_NONNULL RzILVM *vm, RZ_NONNULL RzILOpEffect
 	rz_return_val_if_fail(vm && op, false);
 	return eval_effect(vm, op);
 }
+
+/**
+ * Evaluate the given pure op, return the float
+ * \return NULL if an error occured
+ */
+RZ_API RZ_NULLABLE RZ_OWN RzFloat *rz_il_evaluate_float(RZ_NONNULL RzILVM *vm, RZ_NONNULL RzILOpPure *op) {
+	rz_return_val_if_fail(vm && op, NULL);
+
+	// TODO implement evaluate float
```

Comment:
Does `eval_pure` evaluate it? Then this `TODO` is unnecessary anymore, right?

Reply:
well, I should check. Some TODOs in the comment are implemented but forget to remove them